api-docs/router-dashboard/get-neighbors.md

```diff
@@ -17,19 +17,23 @@ GET
 ### Content
 
 ```
-   [
-   	{
-   		"nickname": "fd00::2",
-   		"route_metric_to_exit": 0,
-   		"total_payments": 0,
-   		"debt": 0
-   	},
-   	{
-   		"nickname": "fd00::7",
-   		"route_metric_to_exit": 0,
-   		"total_payments": 0,
-   		"debt": 0
-   	}
+[
```

Comment:
So what doc convention are rolling with? Is there any reason why #97 can't go in first and for other api-touching PRs to follow suit?

Reply:
sure I guess it could, I just pulled it over here for ease of review.